tethys_gizmos/static/tethys_gizmos/js/cesium_map_view.js

```diff
@@ -314,16 +322,23 @@ var CESIUM_MAP_VIEW = (function() {
             return;
         }
         for (let primitive of m_primitives) {
-            // Cesium Data is stored in options attribute
-            let primitive_option = primitive.options
+            let primitive_option;
+            // Cesium Data is stored in options attribute for MV Layer
+            if (primitive.options) {
+                primitive_option = primitive.options
+            }
+            else {
+                primitive_option = cesium_options(Object.values(primitive)[0]);
```

Comment:
Same comment as above. Is `Object.values(...` needed?

Reply:
Yes, because the key of the primitive option is not part of the cesium data structure.
{'Cesium_OSM_Buildings': {'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 96188}}}}
We don't use the key part to create the cesium object.

Reply:
and primitive.options will have the same data structure as Object.values(primitive)[0]. 
primitives = [MVLayer(source='CesiumPrimitive', legend_title='Cesium 3D Buildings',
                                  options={'Cesium.Cesium3DTileset': {'url': {'Cesium.IonResource.fromAssetId': 96188}}},
                                  data={'layer_name': 'Cesium_Buildings', 'layer_variable': 'variable', 'layer_id': 1})]